src/platform/packages/shared/kbn-esql-ast/src/commands_registry/commands/sort/validate.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+import type { ESQLCommand, ESQLMessage, ESQLAst } from '../../../types';
+import type { ICommandContext } from '../../types';
+import { validateCommandArguments } from '../../../definitions/utils/validation';
+
+export const validate = (
+  command: ESQLCommand,
+  ast: ESQLAst,
+  context?: ICommandContext
+): ESQLMessage[] => {
+  const messages: ESQLMessage[] = [];
+  messages.push(
```

Comment:
Reusing the orphan logic were applicable inside the command validate function

Reply:
Nice  that we are calling now `validateCommandArguments` explicitly 👍